### DISCUSSION TITLE
Web3Modal.configure function

### DISCUSSION THREAD

**Discussion by rebeloper** [WalletConnect/walletconnect-monorepo#3974] [Developer Support]
Hello, we are building a native iOS app with swift and noticed the latest update on mobile integrations. Who can we reach out to about that? We could not find any information that explains "projectId:" and "chainId:" for the Web3Modal.configure function.

**Comment by glitch-txs** [comment]
hi, please open an issue in the web3modal swift repository. Thanks!
